sdks/go/examples/native_wordcap/wordcap.go

```diff
@@ -50,6 +49,11 @@ var (
 	}
 )
 
+// FromBytes converts a byte slice to a string.
+func FromBytes(b []byte) string {
```

Comment:
Is there a reason you fully redefined the function here and then put it in-line in the ParDo?

Reply:
Nope - bad edit 😳  Good catch, fixed